src/components/form.jsx

```diff
@@ -108,20 +108,21 @@ const Form = () => {
                 }
               />
             </div>
-
-            <button
-              onClick={fetchData}
-              disabled={isLoading || !hasUserInputChanged}
-              className="bg-deeppurple text-ash font-bold rounded-xl hover:scale-110 p-2 mt-2 md:mt-0">
-              Search
-            </button>
+            <div className="flex gap-2 md:lg:flex-none ">
```

Comment:
I see that you added a `flex gap-2 md:lg:flex-none` here. The flex is applied only to the button, maybe that's why the change isn't working for me (?) The flex has to be somehow applied to the button and the search result.

Did the changes get applied to your browser though? It'll be cool if you add screenshots to the pull request.